uni/lib/redux/reducers.dart

```diff
@@ -89,6 +90,11 @@ AppState login(AppState state, SaveLoginDataAction action) {
 
 AppState setLoginStatus(AppState state, SetLoginStatusAction action) {
   Logger().i('setting login status: ${action.status}');
+  if (action.status == RequestStatus.successful){
+    Future.delayed(const Duration(seconds: 20), ()=>{
```

Comment:
Why these 20 seconds? Is something else dependant on this initialization?

Reply:
20 seconds is because SharedPreferences doesn't commit the changes to the filesystem immediately, and therefore, other isolates (in this case, in android) will not have the updated version (with the user's username and password) when the app launches for the first time, to be able to login. This is not an optimal way to do it, but I accept suggestions on how to fix it.